starter-core/src/main/java/io/micronaut/starter/feature/github/workflows/docker/templates/mavenGraalCEWorkflow.rocker.raw

```diff
@@ -0,0 +1,42 @@
+@import io.micronaut.starter.application.Project
+
+@args (
+Project project,
+String graalVersion
+)
+
+name: GraalVM CE CI
+on: [push, pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@@v2
+      - uses: actions/cache@@v2
+        with:
+          path: |
+            ~/.m2
+          key: ${{ runner.os }}-maven-test-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-test-
+      - name: Setup GraalVM CE
+        uses: DeLaGuardo/setup-graalvm@@2.0
+        with:
+            graalvm-version: @graalVersion
+      - name: Install Native Image
+        run: gu install native-image
+      - name: Docker login
+        uses: docker/login-action@@v1
+        with:
+          registry: ${{ secrets.DOCKER_REGISTRY_URL }}
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      - name: Build and push docker image
+        env:
+          DOCKER_REGISTRY: ${{ secrets.DOCKER_REGISTRY }}
+        run: |
+          ./mvnw package -Dpackaging=docker-native
+          PROJECT_VERSION=`mvn help:evaluate -Dexpression=project.version -q -DforceStdout`
+          IMAGE="${DOCKER_REGISTRY}/@project.getName():${PROJECT_VERSION}"
```

Comment:
Trying here to stay compatible with gradle version, where docker image with version tag is pushed, meaning latest is not pushed at all.

Reply:
Also once https://github.com/micronaut-projects/micronaut-maven-plugin/issues/48 and https://github.com/micronaut-projects/micronaut-maven-plugin/issues/49 are fixed, the bash part can be removed.